src/main/kotlin/uk/gov/dluhc/printapi/dto/EroManagementApiEroDto.kt

```diff
@@ -3,5 +3,8 @@ package uk.gov.dluhc.printapi.dto
 data class EroManagementApiEroDto(
     val id: String,
     val name: String,
-    val localAuthorities: List<EroManagementApiLocalAuthorityDto>
+    val localAuthorities: List<EroManagementApiLocalAuthorityDto>,
+
+    val englishContactDetails: EroContactDetailsDto,
```

Comment:
This class _may_ change as and when we have changed `ero-management-api` to return ERO Contact and Address data. Until we have designed the response data for that API this will do for now